Britannia/localization/english/replace/BTWK_game_rules_replace_l_english.yml

```diff
@@ -10,4 +10,11 @@
  setting_no_end_date_end_date:1 "#high No End Date#!"
  setting_no_end_date_end_date_desc:0 "The game will not end at any specific given date."
 
-#TODO - no mongol invasion, no hungarian migration, no fall of andalusia
+ #TODO - no mongol invasion, no hungarian migration, no fall of andalusia
+
+ #THIS IS NEW GAME RULE. CAN MOVE TO NEW LOC FILE IF NEEDED GIVEN IS ISN'T A REPLACEMENT
+ rule_enable_BTWK_storyline:0 "#color_dark_green BTWK: Storyline Flavour #"
```

Comment:
Maybe "Storyline Events" - the events are too impactful to be described as mere flavour IMO

Reply:
Oh and given that it's not a replacement it should definitely be moved out of the replace folder

Reply:
What a schoolboy error to make! You definitely wouldn't see me doing something silly like that nowadays! (like _today_)